include/pru/am335x.h

```diff
@@ -42,7 +42,7 @@
 #define __IEP_BASE			0x0002E000
 #define __ECAP0_BASE			0x00030000
 #define __MII_RT_CFG_BASE		0x00032000
-#define __MII_MDIO_BASE			0x00034000
+#define __MII_MDIO_BASE			0x00032400
```

Comment:
Ideally, this should have been a separate commit. If you have time please push as a different change.

Reply:
Done, please check the commits